x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/string/Locate.java

```diff
@@ -80,6 +81,11 @@ protected Pipe makePipe() {
         return NodeInfo.create(this, Locate::new, pattern, input, start);
     }
 
+    @Override
+    public Nullability nullable() {
+        return Nullability.FALSE;
```

Comment:
If the function returns null if any argument is null, this should be `TRUE` not `FALSE`.

Reply:
`LOCATE` does not return `NULL` if `start=NULL`.

Reply:
What about the other parameters?

Reply:
If any of the `pattern` or `input` is `NULL, the result is `NULL`. Only `start` is special.

Reply:
Then the function is nullable as oppose to never nullable. Depending on the params (outside start) it can TRUE or UKNOWN - see `Nullability.and`.

Reply:
You are right, changed.